napari/_vendor/qt_json_builder/qt_jsonschema_form/widgets.py

```diff
@@ -599,6 +600,21 @@ def configure(self):
         self.setGraphicsEffect(self.opacity)
         self.opacity.setOpacity(1)
 
+class DaskSettingsWidget(SchemaWidgetMixin, MagicDaskSettingsWidget):
+
+    @state_property
+    def state(self) -> dict:
+        return self._widget()
+
+    @state.setter
+    def state(self, state: dict):
+        self._widget.dask_enabled.value = state['enabled']
+        self._widget.cache.value = state['cache']
+        
+    def configure(self):
+        self._widget.changed.connect(lambda: self.on_changed.emit(self.state))
+        self._widget.cache.max = self.schema['allOf'][0]['properties']['cache']['maximum']
+
```

Comment:
why it is here, not in `MagicDaskSettingsWidget`?

Reply:
I am not sure how to answer this, except that I followed a similar pattern to every other widget being used in the preferences dialog.  Should it be moved?  @goanpeca do you have any thoughts on this?

Reply:
In `qt_jsonschema_form` the code should be generic, not specific.